lib/venue.rb

```diff
@@ -1,2 +1,40 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+
+    @patrons = []
+  end
+
+
+  def add_patron(patron)
+    patrons << patron
+  end
+
+
+  def yell_at_patrons
+  upcase_patrons = []
+    patrons.each do |patron|
+      upcase_patrons << patron.upcase
+    end
+  upcase_patrons
+  end
+
+  def over_capacity
```

Comment:
Make sure to pay close attention to the interaction pattern. The `over_capacity` method should have a `?` at the end
```
pry(main)> venue.over_capacity?
=> false
```